collect-structure.rst

```diff
@@ -0,0 +1,98 @@
+***********************************************************
+Structure of ODK Collect directory in Android file system
+***********************************************************
+
+After installation of ODK collect, you can try out the app by :ref:`downloading blank example forms <in-app-get-blank-forms>`, as soon as you download a blank form you would be able see odk directory in the internal storage or external storage depending on the settings of your android device. The structure of the directory is explained below.
```

Comment:
this setup seems superfluous, as ot specofis a time when the user might want to go look at the file structure (after "trying out" the app). but looking into the file structure is not intrinsically time-bound to "trying out" the app. -- 

It would be better to just state that blank forms, form instances, and metadata are stored on the dvice, and here is how they are organized.